portfolio/src/main/webapp/script.js

```diff
@@ -189,19 +180,23 @@ function createMap() {
               stylers: [{color: '#17263c'}]
             }
           ]});
+  var options3 = {
```

Comment:
Ideally - its good to avoid variable names such as options3, map1 etc. More descriptive names would be much clearer for readers. 

Ex: tajMahalMarker, tajMahalMap etc.

Its optional for this PR but a note for future code.

Reply:
Noted1.